client/lib/translator-jumpstart/index.jsx

```diff
@@ -69,12 +70,12 @@ communityTranslatorJumpstart = {
 
 		return true;
 	},
-	isActivated: function() {
+	isActivated() {
 		return _shouldWrapTranslations;
 	},
 
-	wrapTranslation: function( originalFromPage, displayedTranslationFromPage, optionsFromPage ) {
-		var props, dataElement;
+	wrapTranslation( originalFromPage, displayedTranslationFromPage, optionsFromPage ) {
+		var props;
```

Comment:
also noting that the logic from lines 87 to 94 could be simplified if we return early

```js
if ( 'string' !== typeof originalFromPage ) {
	debug( 'unknown original format' );
 	return displayedTranslationFromPage;
}

const props = {
	className: 'translatable',
	'data-singular': originalFromPage
};
```